coordinator/src/service.rs

```diff
@@ -583,7 +583,7 @@ impl CoordService {
             Vec::new();
         let request = WriteDataRequest {
             precision: precision as u32,
-            data: Arc::unwrap_or_clone(points.clone()),
+            data: (*points).clone(),
```

Comment:
Arc::unwrap_or_clone() is stable now in toolchain v1.76, to avoid coping `points` if the strong ref count is only 1. Here is the example:

```rust
let request = WriteDataRequest {
    precision: precision as u32,
    data: Arc::unwrap_or_clone(points),
};
``